apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -351,6 +359,19 @@ export class BookingsController {
     return req as unknown as NextApiRequest & { userId?: number } & OAuthRequestParams;
   }
 
+  private async createNextApiRecurringBookingRequest(
+    req: BookingRequest,
+    oAuthClientId?: string,
+    platformBookingLocation?: string
+  ): Promise<NextApiRequest & { userId?: number } & OAuthRequestParams> {
+    const userId = (await this.getOwnerId(req)) ?? -1;
+    const oAuthParams = oAuthClientId
+      ? await this.getOAuthClientsParams(oAuthClientId)
+      : DEFAULT_PLATFORM_PARAMS;
+    Object.assign(req, { userId, ...oAuthParams, platformBookingLocation });
+    return req as unknown as NextApiRequest & { userId?: number } & OAuthRequestParams;
+  }
```

Comment:
can we reuse createNextApiBookingRequest ? I don't see a difference between the too. also this function does not pass noEmail to the body ?